framework/src/main/java/org/checkerframework/common/basetype/BaseTypeChecker.java

```diff
@@ -599,6 +600,8 @@ private void printStoredMessages(CompilationUnitTree unit) {
         final String message;
         /** The source code that the message is about. */
         final @InternedDistinct Tree source;
+        /** Stores the stack trace till the point the checker encounters an error. */
```

Comment:
This is the stack trace when it encountered an error, so this should be something like `The stack trace when the message is created.`
Also change the parameter documentation to the same.

Reply:
I have changed the field and parameter description in my latest commit.